web_src/js/index.js

```diff
@@ -2891,6 +2888,12 @@ function initVueComponents() {
           return 'octicon-repo-forked';
         } if (repo.mirror) {
           return 'octicon-repo-clone';
+        } if (repo.template) {
+          let octicon = 'octicon-repo-template';
```

Comment:
```js
return `octicon-repo-template${repo.private ? '-private': ''}`;
```

Reply:
Ah, yes. Sometimes I forget these shortcuts exist in other languages.